### ISSUE TITLE
2034 doesn't detect variables used in substring expansion

### ISSUE THREAD

**Issue by ian-kelling** [koalaman/shellcheck#293]
```
#!/bin/bash
x=1
y=abc
echo ${y:x}
```

x=1
^-- SC2034: x appears unused. Verify it or export it.

**Comment by LukeShu** [comment]
This looks like a "dup" of #718 .

**Comment by brother** [comment]
Indeed. And as suspected there are no warnings on shellcheck.net for this.

Should be closed.
